src/Koffee.Tests/UtilityTests_Format.fs

```diff
@@ -0,0 +1,57 @@
+module Koffee.UtilityTests_Format
+
+open NUnit.Framework
+open FsUnitTyped
+
+type FakeEnvVarProvider() =
+    interface Format.ISubstitutionProvider with
+        member _.getSubstitution key =
+            match key with
+            | "var" -> Some "REPLACED"
+            | _ -> None
+
+let fakeEnvVarProvider = new FakeEnvVarProvider()
+let subFakeVars = Format.subVars fakeEnvVarProvider
+
+[<Test>]
+let ``Replaces %var% in string`` () =
```

Comment:
Need test for multiple variables in the string.
Would be good with also a test where `%var%%` →`REPLACED%`

Reply:
Added.